app/src/main/java/org/mozilla/fenix/collections/CollectionCreationStore.kt

```diff
@@ -40,7 +39,8 @@ data class CollectionCreationState(
     val selectedTabs: Set<Tab> = emptySet(),
     val saveCollectionStep: SaveCollectionStep = SaveCollectionStep.SelectTabs,
     val tabCollections: List<TabCollection> = emptyList(),
-    val selectedTabCollection: TabCollection? = null
+    val selectedTabCollection: TabCollection? = null,
+    val defaultCollectionNumber: Int = 0
```

Comment:
I think this should start at 1 instead of 0. Most users would expect the count to be one-based not zero-based :)

Reply:
@sblatz , you're right. Changed to 1, thanks!